cli/internal/daemonclient/daemonclient.go

```diff
@@ -0,0 +1,53 @@
+// Package daemonclient is a wrapper around a grpc client
+// to talk to turbod
+package daemonclient
+
+import (
+	"context"
+
+	"github.com/vercel/turborepo/cli/internal/server"
+)
+
+// DaemonClient provides access to higher-level functionality from the daemon to a turbo run.
+type DaemonClient struct {
+	client server.TurboClient
+	ctx    context.Context
+}
+
+// New creates a new instance of a DaemonClient.
+func New(ctx context.Context, client server.TurboClient) *DaemonClient {
+	return &DaemonClient{
+		client: client,
+		ctx:    ctx,
+	}
+}
+
+// GetChangedOutputs implements runcache.OutputWatcher.GetChangedOutputs
+func (d *DaemonClient) GetChangedOutputs(hash string, repoRelativeOutputGlobs []string) ([]string, error) {
+	reply, err := d.client.GetChangedOutputs(d.ctx, &server.GetChangedOutputsRequest{
```

Comment:
This is `reply`, and below you used `resp`. Make 'em consistent? (Don't care which.)

Reply:
Done.